esp/templates/users/profiles/studentinfo.html

```diff
@@ -65,7 +65,7 @@ <h3>Additional Student Information</h3>
 <div class="control-group">
     <label for="id_k12school" class="control-label">School:</label>
     <div class="controls controls-row">
-    <span style="font-size: 0.8em; font-style: italic;">(Begin to type your school's name and select your school when it comes up.)</span>
+    <span style="font-size: 0.8em; font-style: italic;">(Begin to type your school's name and select your school if it comes up, otherwise enter the full school name.)</span>
```

Comment:
```suggestion
    <p style="font-size: 0.8em; font-style: italic;">(Begin to type your school's name and select your school if it comes up, otherwise enter the full school name.)</p>
```